pkg/interactive-installer/.cargo/config.toml

```diff
@@ -0,0 +1,9 @@
+[source.crates-io]
+replace-with = "vendored-sources"
+
+[source."git+https://github.com/gyscos/cursive.git"]
+git = "https://github.com/gyscos/cursive.git"
+replace-with = "vendored-sources"
+
+[source.vendored-sources]
```

Comment:
We do this to avoid having networking in the container. Do you know of a different approach?

Reply:
@jsfakian use ADD directive

Reply:
I could not make it work without networking. As we agreed in the Slack chat, I compressed the vendor directory and extracted it into a Dockerfile. The compressed file is ~250MB, so I had to split it.

Reply:
Brief comment as I'm away from laptop for a while.

We use vendoring not just for reproducibility, but also for SBOM. If network is enabled, and anything can run any command that downloads from the internet, then we cannot reliably reproduce the package (although both go and rust are better about this with hashes), but we also can miss things with the SBOM. That creates a large number of problems. 

When network is no, ADD still works. It is a much more restricted method, hence our dockerfile add scanner can find things and add them to the SBOM.

If it is something more basic - like rust - we should add it upstream to alpine or some other intermediate tooling package, and inherit it from there.

Reply:
@deitch, I did not enable networking in the container. I just compressed the vendor directory in the GitHub repo (at @rucoder's suggestion) to avoid having 200K+ files.

Reply:
Ah, I see this comment now. We need to describe what the process is, so the next person can do it.

If I understand what you are saying, you have the dependencies vendored, but it creates 200k+ files that you don't want to commit to git, so you created a few tar files?

I am not sure committing 200k files to git is terrible (ok, maybe it is 😬 ), but do we even actually need to commit them? Could we exclude the vendored files from git, but require the user to vendor them before invoking a build? I don't know for sure how that would work with our build pipeline, but worth looking at.

FWIW, @mikem-zed started digging more deeply into vendoring, asking the question, "do we really need it??" It won't change in time for this PR (I think), but just because we always did it that way for good reasons, doesn't mean there might not be better ways to comply with the requirements.